packages/databyss-ui/components/Editor/state/reducer.js

```diff
@@ -64,6 +67,74 @@ const setActiveBlockType = (state, type, isNew) => {
   }
 }
 
+const insertBlock = (state, payload) => {
+  const {
+    activeBlockId,
```

Comment:
Let's rename these 2 `insertedBlockId` and `insertedBlockHtml` so they're not confused with `state.activeBlockId`

Then we should add an invariant assertion like:
`invariant(insertedBlockId === state.activeBlockId, 'insertedBlockId must match activeBlockId. It's possible that you called insertNewActiveBlock before activeBlockId was updated')`

Reply:
I'll do this one